corehq/apps/locations/views.py

```diff
@@ -1055,7 +1055,7 @@ def get(self, request, *args, **kwargs):
             next_url = reverse(FilteredLocationDownload.urlname, args=[self.domain])
             next_url_text = _("Go back to organization download")
         else:
-            next_url = reverse("location_export", args=[self.domain])
+            next_url = reverse("manage_locations", args=[self.domain])
```

Comment:
When possible, we try to reference constants so it's easier to trace.  Here's an example that does what I'm describing:

https://github.com/dimagi/commcare-hq/blob/8072e7b68681cd7d2c4567c036d6e32cd2b7931b/corehq/apps/locations/views.py#L171-L171

Can you update to this format?

Reply:
Just changed it. Thanks for the heads up!